src/handlers/profile.ts

```diff
@@ -21,6 +30,25 @@ export default function profileHandler(sessionStore: ISessionStore) {
       return;
     }
 
+    if (options && options.refetch) {
+      if (!session.accessToken) {
+        throw new Error('The access token needs to be saved in the session for the user to be fetched');
+      }
+
+      const { body: user } = await getAsync({
```

Comment:
All OIDC integrations should ideally be called through the OIDC client.

Could you do this instead:

```js
const client = await clientProvider();
const userInfo = await client.userinfo(access_token);
```

Here is an example in the callback handler: https://github.com/auth0/nextjs-auth0/blob/master/src/handlers/callback.ts#L37